BabbleApp/requirements.txt

```
@@ -1,6 +1,6 @@
-onnxruntime==1.19.2; 
+onnxruntime==1.20.1; 
 onnxruntime-directml==1.19.2; platform_system == "Windows"
-onnxruntime-gpu==1.19.2; platform_system == "Linux"
+onnxruntime-gpu==1.20.1; platform_system == "Linux"
```

```suggestion
onnxruntime-gpu==1.92.2; platform_system == "Linux"
```

Currently, `onnxruntime-directml`/`onnxruntime-gpu` wheels do not exist for Python 3.13. As such, this needs to remain on 1.19.2. See: https://github.com/Project-Babble/ProjectBabble/issues/87